app/lib/xmlJsonHelper.js

```diff
@@ -0,0 +1,13 @@
+// the XML2Json parser cannot distinguish between a single tag and what is
+// usually a list of tags but happens to only have one item this makes sure
+// an array is an array
+function asArray(value) {
```

Comment:
This function seems more generic than being in a file called `xmlJsonHelper`. It might be easier for others to find and therefore reuse if the file was called something like `utils` (which isn't a great name but is something I would have a look in to find this type of function).